metaflow/plugins/unbounded_foreach_decorator.py

```diff
@@ -0,0 +1,135 @@
+import os
+import subprocess
+import sys
+from metaflow.cli_args import cli_args
+from metaflow.decorators import StepDecorator
+from metaflow.exception import MetaflowException
+from metaflow.unbounded_foreach import UnboundedForeachInput, UBF_CONTROL, UBF_TASK
+from metaflow.util import to_unicode
+
+class InternalUnboundedForeachInput(UnboundedForeachInput):
```

Comment:
Should we rename this module to `unbounded_foreach_test_decorator.py`? The class names in this module could include the word `test` too since these are only needed to enable tests, they have no other function.

The current name implies that this module is actually required by UBF.

Reply:
Sure. It was named that way in #450. @savingoyal changed it to this. I have no preference either way.

Reply:
Called in InternalTest... and changed other references.